builder/dockerfile/parser/testfiles/continueIndent/Dockerfile

```diff
@@ -30,7 +30,5 @@ frog
 
 RUN echo hello \
 # this is a comment
-
 # this is a comment with a blank line surrounding it
```

Comment:
yes this was intentionally weird and while I'm 100% in favor of "fixing" this, it needs to be discussed by the maintainers because this was purposely kept weird all this time.